### PR TITLE
Thk fixes

### PR DESCRIPTION
**Pull request by TomKimsey**

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # about: Send an email containing a survey upon topic solved
 # version: 0.1
 # authors: HMS Solution Center Americas
-# url: https://github.com/HMSAB/discourse-survey-monkey.git
+# url: https://github.com/hms-networks/discourse-survey.git
```
Comment (TomKimsey) [inline plugin.rb]: I think I forgot to remove the .git at the end of the url. I will do that when i push any changes you request @oliver-walker-hms

File: config/locales/server.en.yml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 en:
   site_settings:
-    survey_monkey_enabled: "Enable to enable survey monkey plugin."
-    survey_monkey_survey_url: "Set to your specific survey monkey survey."
-    survey_monkey_manual_button: "Enable to add a button to topics to manually force send a survey"
-  survey_monkey:
+    survey_enabled: "Enable to enable survey plugin."
```
Comment (oliver-walker-hms) [inline config/locales/server.en.yml:3]: Enable to enable has an odd ring to it
```suggestion
    survey_enabled: "Enables survey plugin."
```

File: config/locales/server.en.yml [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
+    survey_survey_url: "Set to your specific survey survey."
```
Comment (oliver-walker-hms) [inline config/locales/server.en.yml:4]: Jordan liked to repeat things
```suggestion
    survey_survey_url: "Set to your survey URL."
```

**Review by oliver-walker-hms — CHANGES_REQUESTED** [review]

File: config/locales/server.en.yml (hunk 3 above)
Comment (TomKimsey) [reply config/locales/server.en.yml:4]: Not within the scope of this commit, will change in next pr.

File: config/locales/server.en.yml (hunk 2 above)
Comment (TomKimsey) [reply config/locales/server.en.yml:3]: Not within the scope of this commit, will change in next pr.

File: plugin.rb (hunk 1 above)
Comment (oliver-walker-hms) [reply plugin.rb]: I didn't catch anything else so after you push that i'll approve it

**Review by oliver-walker-hms — APPROVED** [review]

**Pull request merged**